### PR TITLE
ColorPicker: Fix opacity slider and improve demo

### LINKED ISSUE #7073: ColorPicker: Opacity slider changing when used
### Environment Information 
 
 
- __Package version(s)__: Latest
- __Browser and OS versions__: All

### Please provide a reproduction of the bug in a codepen:
This bug can be seen on the Fabric website: https://developer.microsoft.com/en-us/fabric#/components/colorpicker
 
#### Actual behavior:
See [this GIF](https://duaw26jehqd4r.cloudfront.net/items/2V1Q0R1l1Y2U2o153s0D/Screen%20Recording%202018-11-11%20at%2001.09%20PM.gif?v=69a4f55e). The color at the right side of the slider should always be at 100% opacity, but it's changing to reflect the selected value.

#### Expected behavior:
 
The right side of the slider should always be at 100% opacity.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? Yes
 
Requested priority: Low

### PR DESCRIPTION
**Pull request by mikewheaton**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #7073
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Fixes a bug where the opacity slider was not showing the correct value on the right side (see #7073). I've also added a selected color swatch to the example page to preview the final color and demonstrate how to use the component.

**Before**:
image
image

**After**:
image
image

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/ColorPicker/examples/ColorPicker.Basic.Example.tsx [hunk 1]
```diff
@@ -1,12 +1,33 @@
...
+
+    this._updateColor = this._updateColor.bind(this);
+  }
+
   public render(): JSX.Element {
-    return <ColorPicker color="#FFFFFF" />;
+    return (
+      <div style={{ display: 'flex' }}>
+        <ColorPicker color={this.state.color} onColorChanged={this._updateColor} />
+        <div style={{ backgroundColor: this.state.color, width: 100, height: 100, margin: 16, border: '1px solid #c8c6c4' }} />
```
Comment (kenotron) [inline packages/office-ui-fabric-react/src/components/ColorPicker/examples/ColorPicker.Basic.Example.tsx]: Since this is an example, maybe we can encourage the use of mergeStyles here instead of inline styles?

File: packages/office-ui-fabric-react/src/components/ColorPicker/examples/ColorPicker.Basic.Example.tsx [hunk 2]
```diff
@@ -1,12 +1,33 @@
-import * as React from 'react';
 import { ColorPicker } from 'office-ui-fabric-react/lib/ColorPicker';
+import * as React from 'react';
```
Comment (kenotron) [inline packages/office-ui-fabric-react/src/components/ColorPicker/examples/ColorPicker.Basic.Example.tsx]: Is there a specific reason to swap order of this import?

File: packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.base.tsx [hunk 3]
```diff
@@ -1,10 +1,11 @@
 import * as React from 'react';
+
+import { ITextField, TextField } from '../../TextField';
 import { BaseComponent, classNamesFunction } from '../../Utilities';
+import { getColorFromRGBA, getColorFromString, IColor, MAX_COLOR_HUE, updateA, updateH, updateSV } from '../../utilities/color/colors';
```
Comment (kenotron) [inline packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.base.tsx]: These changes to imports are pretty simple, but they're pretty hard to parse as humans since the changes aren't needed (there's no changes, right?). Maybe there's a prettier / plugin that's doing these re-orders for you?

Comment (mikewheaton) [reply packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.base.tsx]: Thanks for catching this, it's caused by the TypeScript Hero extension I have running. This is fine for new files but there's no reason to introduce unrelated changes in PRs. I'll revert these. 😄

File: packages/office-ui-fabric-react/src/components/ColorPicker/examples/ColorPicker.Basic.Example.tsx (hunk 1 above)
Comment (mikewheaton) [reply packages/office-ui-fabric-react/src/components/ColorPicker/examples/ColorPicker.Basic.Example.tsx]: Is there an easy way to apply mergeStyles to a div without having to create interfaces, a `*.base.tsx` file, and so on? I can add that, it just feels like overkill for a simple component.

Comment (mikewheaton) [reply packages/office-ui-fabric-react/src/components/ColorPicker/examples/ColorPicker.Basic.Example.tsx]: This is good feedback on our examples, as I can't find any that use mergeStyles and this could be an opportunity to demonstrate it. But I think it's out of scope for this PR given that it matches what we do in other examples.
